index.html

```diff
@@ -0,0 +1,997 @@
+          <button class="btn-primary-lg">Primary Button</button>
+          <button class="btn-primary-md mg-top-half mg-bottom-half">Primary Button</button>
+          <button class="btn-primary-sm">Primary Button</button>
+        </div>        
+
+        <!-- <pre>          &#x3C;button class=&#x22;btn-primary-lg&#x22;&#x3E;Primary Button&#x3C;/button&#x3E;
+          &#x3C;button class=&#x22;btn-primary-md&#x22;&#x3E;Primary Button&#x3C;/button&#x3E;
+          &#x3C;button class=&#x22;btn-primary-sm&#x22;&#x3E;Primary Button&#x3C;/button&#x3E;
+        </pre> -->
+        <br/>
+        <br/>
+        <div class="code">
+          <pre>
+            <code class="language-html">
+              &#x3C;button class=&#x22;btn-primary-lg&#x22;&#x3E;Primary Button&#x3C;/button&#x3E;
+              &#x3C;button class=&#x22;btn-primary-md&#x22;&#x3E;Primary Button&#x3C;/button&#x3E;
+              &#x3C;button class=&#x22;btn-primary-sm&#x22;&#x3E;Primary Button&#x3C;/button&#x3E;    
+            </code>
+          </pre>
+        </div>
+        
+        <br/>
+        <div class="row-responsive ">
+          <button class="btn-secondary-lg">secondary Button</button>
+          <button class="btn-secondary-md mg-top-half mg-bottom-half">secondary Button</button>
+          <button class="btn-secondary-sm">secondary Button</button>
+        </div>
+
+       
+        <br/>
+        <br/>
+        <div class="code">
+          <pre>
+            <code class="language-html">
+              &#x3C;button class=&#x22;btn-secondary-lg&#x22;&#x3E;secondary Button&#x3C;/button&#x3E;
+              &#x3C;button class=&#x22;btn-secondary-md&#x22;&#x3E;secondary Button&#x3C;/button&#x3E;
+              &#x3C;button class=&#x22;btn-secondary-sm&#x22;&#x3E;secondary Button&#x3C;/button&#x3E;
+            </code>
+          </pre>
+        </div>
+        
+        <br/>
+        <br/>
+        <br/>
+        <a href="..." class="btn-primary-md">Link Button</a>
+        <a href="..." class="btn-secondary-md">Link Button</a>
+        <br/>
+        <br/>
+        <div class="code">
+          <pre>
+            <code class="language-html">
+              &#x3C;a href=&#x22;&#x22; class=&#x22;btn-primary-md&#x22;&#x3E;Link Button&#x3C;/a&#x3E;
+              &#x3C;a href=&#x22;&#x22; class=&#x22;btn-secondary-md&#x22;&#x3E;Link Button&#x3C;/a&#x3E;
+            </code>
+          </pre>
+        </div>
+        <div class="relative">
+          <button class="btn-floating relative ">+</button>
+        </div>
+        floating action button(fab)
+        <br/>
+        <br/>
+      </div>
+      <div class="pd-1 mg-bottom-1" id="card">
+        <h3>Card</h3>
+        <div class="card" style="width:18rem">
+            <img class="card-image" src="https://cdn.discordapp.com/avatars/778699572787675136/bfd4785c0eb8a2be4eae7cc2fec27d34.png?size=256&quot"
+          >
+          <div class="card-body">
+            <div class="card-head">Header</div>
```

Comment:
`card-head` seems to need a little bit more emphasis in all cards. Maybe slightly increase the font-weight.